src/shacl-shape-lib/descriptors-shacl-shape.xsl

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xmlns:math="http://www.w3.org/2005/xpath-functions/math"
+    xmlns:xd="http://www.oxygenxml.com/ns/doc/xsl"
+    xmlns:f="http://https://github.com/costezki/model2owl#"
+    xmlns:fn="http://www.w3.org/2005/xpath-functions"
+    exclude-result-prefixes="xs math xd xsl uml xmi fn f"
+    xmlns:uml="http://www.omg.org/spec/UML/20131001"
+    xmlns:xmi="http://www.omg.org/spec/XMI/20131001" xmlns:owl="http://www.w3.org/2002/07/owl#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:sh="http://www.w3.org/ns/shacl#"
+    xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:dc="http://purl.org/dc/elements/1.1/"
+    xmlns:foaf="http://xmlns.com/foaf/0.1/" version="3.0">
+
+    <xd:doc scope="stylesheet">
+        <xd:desc>
+            <xd:p><xd:b>Created on:</xd:b> Mar 21, 2020</xd:p>
+            <xd:p><xd:b>Author:</xd:b> lps</xd:p>
+            <xd:p>This module defines how selected XMI elements are transformed into OWL2
+                statements</xd:p>
+        </xd:desc>
+    </xd:doc>
+
+
+    <xd:doc>
+        <xd:desc>Rule T.02. Label — in data shape layer. Specify a label for the SHACL shape, based on the name of the UML element.</xd:desc>
+        <xd:param name="uri"/>
+        <xd:param name="elementName"/>
+    </xd:doc>
+    <xsl:template name="shapeLayerName">
+        <xsl:param name="elementName"/>
+        <xsl:param name="uri"/>
+        
+        <rdf:Description rdf:about = "{$uri}">
```

Comment:
This is a complete and correct solution for generating a description triplet. apply the same for other descriptor rules in core layer.